ddtrace/contrib/unittest/patch.py

```diff
@@ -51,6 +51,8 @@ def get_version():
 
 
 def _enable_unittest_if_not_started():
+    if not hasattr(_CIVisibility, "_unittest_data"):
+        _CIVisibility._unittest_data = {"suites": {}, "modules": {}}
```

Comment:
is there a reason why we do the `hasattr` call here and add a new property dynamically vs defining `_unittest_data` on the `CIVisibility` in the class definition?

Reply:
`CIVIsibility` isn't inherently tied to `unittest`, so I don't particularly like the idea of having test-framework-related data in there. Putting it in the class definition "codifies" it in a way I don't like.

Doing it this way has its own janky vibe, and very much "half a dozen of one, six of the other".

Either way, I plan to rewrite `CIVisibility` somewhat extensively in Q4 or Q1 anyway, so whatever change we make here will be relatively short-lived.

Reply:
👍🏻